docs/sources/mimir/configure/configuration-parameters/index.md

```diff
@@ -3781,6 +3781,16 @@ kafka:
   # CLI flag: -ingest-storage.kafka.write-clients
   [write_clients: <int> | default = 1]
 
+  # The username to use to authenticate to Kafka using SASL plain mechanism.
```

Comment:
```suggestion
  # The username used to authenticate to Kafka using the SASL plain mechanism.
```

Reply:
Done in https://github.com/grafana/mimir/pull/9584/commits/1b51166f8d35f5f67bbf8c575b999e03ce0276db